server/logging/kinesis_test.go

```diff
@@ -70,6 +72,25 @@ func TestKinesisNormalPut(t *testing.T) {
 	assert.Equal(t, 1, callCount)
 }
 
+//func TestKinesisNormalPut(t *testing.T) {
+//	ctx := context.Background()
+//	callCount := 0
+//	//putFunc := func(input *kinesis.PutRecordsInput) (*kinesis.PutRecordsOutput, error) {
+//	//	callCount += 1
+//	//	assert.Equal(t, logs, getLogsFromPutRecordsInput(input))
+//	//	assert.Equal(t, "foobar", *input.StreamName)
+//	//	return &kinesis.PutRecordsOutput{FailedRecordCount: aws.Int64(0)}, nil
+//	//}
+//	kk, err := NewKinesisLogWriter("us-east-2", "", "", "", "", "test1", log.NewNopLogger())
+//	assert.NoError(t, err)
+//
+//	//k := &mock.KinesisMock{PutRecordsFunc: putFunc}
+//	//writer := makeKinesisWriterWithMock(k, "foobar")
+//	err = kk.Write(ctx, logs)
+//	assert.NoError(t, err)
+//	assert.Equal(t, 1, callCount)
+//}
```

Comment:
What's this? (If this is handy to have commented-out please add a comment at the top on why :)

Reply:
whoops my mistake